.circleci/config.yml

```
@@ -134,12 +134,16 @@ jobs:
       - run:
           command: npm run build
           working_directory: client
+          environment:
+            REACT_APP_GRAPHQL_ENDPOINT: https://zenika.faq.team/gql
+            REACT_APP_REST_ENDPOINT: https://zenika.faq.team/rest
```

J'avais pas pensé sur le coup mais en fait on doit pouvoir faire ça, puisque le front et le back sont sur le même serveur :

```suggestion
            REACT_APP_GRAPHQL_ENDPOINT: /gql
            REACT_APP_REST_ENDPOINT: /rest
```

C'est beaucoup plus générique :)

Tu peux tester ?